lib/api/utilities/functions.php

```diff
@@ -86,11 +86,11 @@ function beans_remove_dir( $dir_path ) {
 		if ( is_dir( $path ) ) {
 			beans_remove_dir( $path );
 		} else {
-			@unlink( $path );
+			@unlink( $path ); // phpcs:disable Generic.PHP.NoSilencedErrors.Discouraged - This is a valid use case.
```

Comment:
Just to note, this requires PHPCS 3.2.0, which is not released yet, but probably will be soon. I don't know if the other standards will have any fixing to do to be able to use it.